packages/web/package.json

```diff
@@ -33,7 +33,6 @@
     "jest-mock-extended": "^2.0.7",
     "postcss": "^8.4.14",
     "prisma": "^4.2.0",
-    "tailwindcss": "^3.1.6",
-    "typescript": "^4.7.4"
```

Comment:
typescript is also declared in dependencies, so this is a dupe that emits a warning in main